chia/full_node/mempool.py

```diff
@@ -37,37 +37,37 @@ def get_min_fee_rate(self, cost: int) -> float:
         else:
             return 0
 
-    def remove_from_pool(self, item: MempoolItem):
+    def remove_from_pool(self, items: List[bytes32]) -> None:
         """
         Removes an item from the mempool.
         """
-        removals: List[Coin] = item.removals
-        additions: List[Coin] = item.additions
-        for rem in removals:
-            del self.removals[rem.name()]
-        for add in additions:
-            del self.additions[add.name()]
-        del self.spends[item.name]
-        del self.sorted_spends[item.fee_per_cost][item.name]
-        dic = self.sorted_spends[item.fee_per_cost]
-        if len(dic.values()) == 0:
-            del self.sorted_spends[item.fee_per_cost]
-        self.total_mempool_cost -= item.cost
-        assert self.total_mempool_cost >= 0
-
-    def add_to_pool(
-        self,
-        item: MempoolItem,
-    ):
+        for spend_bundle_id in items:
+            item = self.spends[spend_bundle_id]
```

Comment:
I'm not sure if this matters, but if `spend_bundle_id` cannot be found in `spends`, this will throw and terminate the loop. would it make sense to at least try to remove the remaining items too?

Reply:
good point, will do that.